app/controllers/api/v1/items_controller.rb

```diff
@@ -83,6 +84,10 @@ def update
       end
     end
 
+    def browse
+      render json: @items.accepted, each_serializer: serializer
+    end
+
```

Comment:
Sure @steveyken 
we will add separate `BrowseController` for browse app. Thanks.